### YamlMime:ManagedReference
items:
- uid: System.Activities.UnhandledExceptionAction
  id: UnhandledExceptionAction
  children:
  - System.Activities.UnhandledExceptionAction.Abort
  - System.Activities.UnhandledExceptionAction.Cancel
  - System.Activities.UnhandledExceptionAction.Terminate
  langs:
  - csharp
  name: UnhandledExceptionAction
  nameWithType: UnhandledExceptionAction
  fullName: System.Activities.UnhandledExceptionAction
  type: Enum
  summary: "Hiermee geeft u de actie die wordt uitgevoerd wanneer een uitzondering verlaat u de hoofdmap van een werkstroom."
  remarks: "De <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A>functie als een uitzondering verlaat u de hoofdmap van de werkstroom wordt aangeroepen.</xref:System.Activities.WorkflowApplication.OnUnhandledException%2A> De <xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs>biedt toegang tot de uitzondering evenals een verwijzing naar de <xref:System.Activities.Activity>die de uitzondering gegenereerd.</xref:System.Activities.Activity> </xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs> Beëindigen is de standaardmethode als er geen <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A>handler is opgegeven.</xref:System.Activities.WorkflowApplication.OnUnhandledException%2A>"
  example:
  - "The following example invokes a workflow that throws an exception. The exception is unhandled by the workflow and the <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A> handler is invoked. The <xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs> are inspected to provide information about the exception, and the workflow is terminated.  \n  \n [!code-cs[CFX_WorkflowApplicationExample#6](~/add/codesnippet/csharp/t-system.activities.unha_1.cs)]"
  syntax:
    content: public enum UnhandledExceptionAction
  inheritance:
  - System.Enum
  platform:
  - net462
- uid: System.Activities.UnhandledExceptionAction.Abort
  id: Abort
  parent: System.Activities.UnhandledExceptionAction
  langs:
  - csharp
  name: Abort
  nameWithType: UnhandledExceptionAction.Abort
  fullName: System.Activities.UnhandledExceptionAction.Abort
  type: Field
  assemblies:
  - System.Activities
  namespace: System.Activities
  summary: "Hiermee wordt aangegeven dat de <xref href=&quot;System.Activities.WorkflowApplication&quot;> </xref> moet de workflow afgebroken."
  remarks: "Dit resulteert in <xref:System.Activities.WorkflowApplication.Aborted%2A>wordt aangeroepen wanneer de abort-proces voltooid is.</xref:System.Activities.WorkflowApplication.Aborted%2A> De niet-verwerkte uitzondering wordt gebruikt als de reden afbreken."
  example:
  - "The following example invokes a workflow that throws an exception. The exception is unhandled by the workflow and the <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A> handler is invoked. The <xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs> are inspected to provide information about the exception, and the workflow is aborted.  \n  \n [!code-cs[CFX_WorkflowApplicationExample#7](~/add/codesnippet/csharp/t-system.activities.unha_2.cs)]"
  syntax:
    content: Abort
    return:
      type: System.Activities.UnhandledExceptionAction
      description: "Moet worden toegevoegd."
  exceptions: []
  platform:
  - net462
- uid: System.Activities.UnhandledExceptionAction.Cancel
  id: Cancel
  parent: System.Activities.UnhandledExceptionAction
  langs:
  - csharp
  name: Cancel
  nameWithType: UnhandledExceptionAction.Cancel
  fullName: System.Activities.UnhandledExceptionAction.Cancel
  type: Field
  assemblies:
  - System.Activities
  namespace: System.Activities
  summary: "Hiermee wordt aangegeven dat de <xref href=&quot;System.Activities.WorkflowApplication&quot;> </xref> moet het annuleren van de hoofdactiviteit planning en uitvoering hervatten."
  remarks: "Dit resulteert in <xref:System.Activities.WorkflowApplication.Completed%2A>wordt aangeroepen wanneer de annuleringsproces voltooid is.</xref:System.Activities.WorkflowApplication.Completed%2A>"
  example:
  - "The following example invokes a workflow that throws an exception. The exception is unhandled by the workflow and the <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A> handler is invoked. The <xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs> are inspected to provide information about the exception, and the workflow is cancelled.  \n  \n [!code-cs[CFX_WorkflowApplicationExample#8](~/add/codesnippet/csharp/t-system.activities.unha_3.cs)]"
  syntax:
    content: Cancel
    return:
      type: System.Activities.UnhandledExceptionAction
      description: "Moet worden toegevoegd."
  exceptions: []
  platform:
  - net462
- uid: System.Activities.UnhandledExceptionAction.Terminate
  id: Terminate
  parent: System.Activities.UnhandledExceptionAction
  langs:
  - csharp
  name: Terminate
  nameWithType: UnhandledExceptionAction.Terminate
  fullName: System.Activities.UnhandledExceptionAction.Terminate
  type: Field
  assemblies:
  - System.Activities
  namespace: System.Activities
  summary: "Hiermee wordt aangegeven dat de <xref href=&quot;System.Activities.WorkflowApplication&quot;> </xref> moet beëindiging van de hoofdactiviteit planning en uitvoering hervatten."
  remarks: "Dit resulteert in <xref:System.Activities.WorkflowApplication.Completed%2A>wordt aangeroepen wanneer de beëindiging-proces voltooid is.</xref:System.Activities.WorkflowApplication.Completed%2A> De niet-verwerkte uitzondering wordt gebruikt als de beëindiging reden. Beëindigen is de standaardmethode als er geen <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A>handler is opgegeven.</xref:System.Activities.WorkflowApplication.OnUnhandledException%2A>"
  example:
  - "The following example invokes a workflow that throws an exception. The exception is unhandled by the workflow and the <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A> handler is invoked. The <xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs> are inspected to provide information about the exception, and the workflow is terminated.  \n  \n [!code-cs[CFX_WorkflowApplicationExample#6](~/add/codesnippet/csharp/t-system.activities.unha_1.cs)]"
  syntax:
    content: Terminate
    return:
      type: System.Activities.UnhandledExceptionAction
      description: "Moet worden toegevoegd."
  exceptions: []
  platform:
  - net462
references:
- uid: System.Enum
  isExternal: false
  name: System.Enum
- uid: System.Activities.UnhandledExceptionAction.Abort
  parent: System.Activities.UnhandledExceptionAction
  isExternal: false
  name: Abort
  nameWithType: UnhandledExceptionAction.Abort
  fullName: System.Activities.UnhandledExceptionAction.Abort
- uid: System.Activities.UnhandledExceptionAction
  parent: System.Activities
  isExternal: false
  name: UnhandledExceptionAction
  nameWithType: UnhandledExceptionAction
  fullName: System.Activities.UnhandledExceptionAction
- uid: System.Activities.UnhandledExceptionAction.Cancel
  parent: System.Activities.UnhandledExceptionAction
  isExternal: false
  name: Cancel
  nameWithType: UnhandledExceptionAction.Cancel
  fullName: System.Activities.UnhandledExceptionAction.Cancel
- uid: System.Activities.UnhandledExceptionAction.Terminate
  parent: System.Activities.UnhandledExceptionAction
  isExternal: false
  name: Terminate
  nameWithType: UnhandledExceptionAction.Terminate
  fullName: System.Activities.UnhandledExceptionAction.Terminate
